pkg/mappings.bzl

```diff
@@ -494,6 +494,15 @@ pkg_mklink = rule(
     implementation = _pkg_mklink_impl,
     # @unsorted-dict-items
     attrs = {
+        "target": attr.string(
+            doc = """Link "target", a path on the filesystem.
+
+            This is what the link "points" to, and may point to an arbitrary
+            filesystem path, even relative paths.
+
+            """,
+            mandatory = True,
+        ),
```

Comment:
While I like that the link target should be called "target", I feel there's a potential bit of confusion with "dest" -- it could refer to the link's destination rather than the destination in the package. 

Maybe "dest" should  be "link_name"?  That follows conventions closer to the `ln` (unix) and `mklink` (windows) commands.

Reply:
Great. I like link_name better too.